common/src/main/java/com/microsoft/hydralab/common/util/RestTemplateConfig.java

```diff
@@ -21,9 +19,7 @@
 import java.security.KeyStoreException;
 import java.security.NoSuchAlgorithmException;
 
-@Configuration
```

Comment:
What's this change about?

Reply:
The bean created by this classis class is never used. The change has been split to another PR.